src/main/webapp/aboutus.html

```diff
@@ -41,11 +41,11 @@ <h2>Teammate B Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate C Name</h2>
+    <h2>Zari Haidarian</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+      <li>Summer Feelz: Working, learning, and traveling</li>
```

Comment:
All the other lines end with a period - I think maybe you wanted to do the same here? It would like nicer if it's consistent I think.